lib-extra/src/test/java/com/diffplug/spotless/extra/java/EclipseJdtFormatterStepTest.java

```diff
@@ -24,7 +24,8 @@
 
 	@Override
 	protected String[] getSupportedVersions() {
-		return new String[]{"4.6.1", "4.6.2", "4.6.3", "4.7.0", "4.7.1", "4.7.2", "4.7.3a", "4.8.0", "4.9.0", "4.10.0"};
+		return new String[]{"4.6.1", "4.6.2", "4.6.3", "4.7.0", "4.7.1", "4.7.2", "4.7.3a", "4.8.0", "4.9.0", "4.10.0",
```

Comment:
The formatting is broken, that's why the build fails.
Run 'gradlew spotlessApply' to fix these violations.

Reply:
Done, it catched a .gradle formatting error too so I included it in the PR

Reply:
:+1: